packages/pn-personafisica-webapp/src/redux/notification/reducers.ts

```diff
@@ -166,28 +200,28 @@ const notificationSlice = createSlice({
           payment.isLoading = true;
           return;
         }
-        state.paymentInfo = [{ ...paymentHistory[0], isLoading: true }];
+        state.paymentsData.pagoPaF24 = [{ ...state.paymentsData.pagoPaF24[0], isLoading: true }];
       }
     });
     builder.addCase(getNotificationPaymentUrl.rejected, (state, action) => {
       const noticeCode = action.meta.arg.paymentNotice.noticeNumber;
       const creditorTaxId = action.meta.arg.paymentNotice.fiscalCode;
-      const paymentInfo = state.paymentInfo.find(
+      const paymentInfo = state.paymentsData.pagoPaF24.find(
         (payment) =>
           payment.pagoPA?.creditorTaxId === creditorTaxId &&
           payment.pagoPA?.noticeCode === noticeCode
       );
 
       if (paymentInfo && paymentInfo.pagoPA) {
         const updatedPaymentInfo: PaymentHistory = {
```

Comment:
Also here, maybe you can update paymentInfo variable directly and remove the row 224.